docs/howto/windows.md

```diff
@@ -79,14 +68,6 @@ You'll want to use the Windows side of things for:
 * the Android emulator
 * the location of your development tree in the filesystem
 
-We're not sure yet which side you'll want to use for the following;
-try them on both the Linux side and the Windows side, and let us know
-which you prefer:
-* the `git` command line (from your `bash` shell either way, but you
-  might have it run the Windows version)
-* Gradle, which is the Java program that does the Android build
-  (again, invoking it from your `bash` shell either way)
-* the JDK, which is used for running Gradle
```

Comment:
Do you have a conclusion on which side you prefer for each of these to run on? If so, what did you learn that led to that conclusion?

More broadly, what's your thinking behind removing this paragraph?

Reply:
I left the paragraph because I simply was unable to get Gradle and the JDK working on the WSL side. I encountered several errors that I was unable to figure out despite some extensive googling. I am still trying to figure it out since I'm unhappy with the number of steps required to get setup when you have to install the JDK and Android Studio on the Windows side and I know if everything was done from WSL I could probably half the steps. But until we have a clear path for a new user to follow, I hesitate to have them spend as many hours as I did and give up halfway.